src/components/Footer.tsx

```diff
@@ -1,29 +1,50 @@
-import React from 'react'
+import React, { useRef, useEffect } from 'react'
 import { connect, useDispatch } from 'react-redux'
 import * as pkg from '../../package.json'
 import { TUTORIAL2_STEP_SUCCESS } from '../constants'
-import { logout, showModal } from '../action-creators'
+import { alert, logout, showModal } from '../action-creators'
 import { getSetting, isTutorial } from '../selectors'
 import { scaleFontDown, scaleFontUp } from '../action-creators/scaleSize'
 import { State } from '../util/initialState'
 
 // eslint-disable-next-line jsdoc/require-jsdoc
 const mapStateToProps = (state: State) => {
-  const { authenticated, status, user } = state
+  const { authenticated, status, user, fontSize } = state
   return {
     authenticated,
     status,
     isTutorialOn: isTutorial(state),
     tutorialStep: +(getSetting(state, 'Tutorial Step') || 1),
     user,
+    fontSize,
   }
 }
 
 /** A footer component with some useful links. */
-const Footer = ({ authenticated, tutorialStep, user, isTutorialOn, status }: ReturnType<typeof mapStateToProps>) => {
+const Footer = ({ authenticated, tutorialStep, user, isTutorialOn, status, fontSize }: ReturnType<typeof mapStateToProps>) => {
 
   const dispatch = useDispatch()
 
+  const firstUpdate = useRef(true)
+
+  useEffect(() => {
+    /** Below check is needed to stop alert dispatch when rendered for first time. */
+    if (!firstUpdate.current) {
+      dispatch(alert(`Font size: ${fontSize}`))
```

Comment:
FYI the `alert` action-creator has a built-in timer with options to clear it after a specific delay. (No expectations for you to have known that.)